{{cookiecutter.project_name}}/Applications/{{cookiecutter.app_name}}App/Resources/Settings/DefaultSettings.ini

```
@@ -1,5 +1,8 @@
 [General]
 
+[ApplicationUpdate]
+ServerUrl=https://download.slicer.org
```

Since the custom app is not related to Slicer, is this needed ?

---

I was just providing an example if a custom app wanted to specify their own url. We could just leave it as empty, but at least include the setting so that developers know which one to update.
```suggestion
ServerUrl=
```

---

I could imagine custom app SlicerSalt could modify it to something based on https://salt.slicer.org/download/